plugins/katello/developers.md

```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
+  * Ready for Testing: An issue has an open PR
+  * Closed: An issue has been fixed
+
+### Release States
+
+There are a few important states the issue can be in with respect to the 'Release' field within Redmine. These release states have an impact not only a release itself but the triage process as a whole. A quick definition of each that will make more sense in the process section:
+
+  * Empty: No release has been set, this is the "untriaged" bucket
+  * Katello Backlog: Issues that have been put on the backlog and not scheduled for a release due to any number of reasons
+  * Katello Recycle Bin: Issues that have been rejected, or are a duplicate, or have been in 'Needs more information' for longer than a week
+  * Katello X.Y.Z: Standard releases for issues to be targeted at
+
+### Process
+
+The triage runner starts the meeting by going through sets of issues broken down into different groups by priority of the grouping. This is so that if time were to run out, the most critical categories of issues are triaged. This list (with easy links for the runner) are at [1]. The runner is in charge of ensuring the flow of the meeting, and updating issues or ensuring that someone on the call takes responsibility for updating an issue. This latter part sometimes occurs if a duplicate needs to be looked up, or another developer on the call can explain/ask the user for more information more clearly than the runner.
+
+### Recycle Bin
+
+The first on the list is our 'recycle bin'. This the place to put issues that don't fit into a release, or were filed by a user who has since gone dark on the issue. Another way to look at it is a place for unresolvable issues. Issues are moved onto the recycle bin if there has been no activity by a user for a week. And they are closed if a week on the recycle bin goes by without any further updates.
+
+#### General Flow for New or Needs More Information
+
+The general flow is to open an issue and the runner to exam the issue, typically reading the general idea outloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answers the following questions:
+
+	* Does this issue need more information from the user?
+	* If yes, leave a comment asking for that information
+	* Does this issue need a developer to investigate it further to properly triage it?
+	* What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+	* Does any developer want to own it?
+	* If yes, assign it
+	* Issue category?
+
+#### General Flow for Ready for Testing
+
+  * Will this PR get completed by the next release?
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+
+#### General Flow for Closed
+
+  * Should this issue be targeted at the next z-stream? Current RC phase? Or just throw it in the "next" release (aka develop or master)
```

Comment:
This is a general flow for any state, it seems. Perhaps we could just say this at the end of this list.

Reply:
I thought about that, but I personally liked thinking about each state respectively.